diff --git a/Jellyfin.Networking/Configuration/NetworkConfiguration.cs b/Jellyfin.Networking/Configuration/NetworkConfiguration.cs
index 361dbc814..b090a1f74 100644
--- a/Jellyfin.Networking/Configuration/NetworkConfiguration.cs
+++ b/Jellyfin.Networking/Configuration/NetworkConfiguration.cs
@@ -115,7 +115,7 @@ namespace Jellyfin.Networking.Configuration
         /// <summary>
         /// Gets or sets a value indicating whether gets or sets IPV6 capability.
         /// </summary>
-        public bool EnableIPV6 { get; set; }
+        public bool EnableIPV6 { get; set; } = true;
 
         /// <summary>
         /// Gets or sets a value indicating whether gets or sets IPV4 capability.
